app/content/models/priority_pool.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+from app.content.models import Event
+from app.group.models import Group
+from app.util.models import BaseModel
+
+
+class PriorityPool(BaseModel):
+
+    groups = models.ManyToManyField(Group, related_name="event_priority_pools")
+    event = models.ForeignKey(
```

Comment:
Denne kan bli lagt til som mange-til-mange fra event til group

Reply:
Kan man ha mange til mange gjennom to modeller? Event -> PriorityPool -> PriorityPoolGroup -> Group